Documentation/ceph-cluster-crd.md

```diff
@@ -1234,6 +1236,8 @@ spec:
     #rulesNamespace: rook-ceph
     #externalMgrEndpoints:
       #- ip: 192.168.39.182
+    # externalMgrServicesPort:
```

Comment:
nit: the other commented examples don't have a space after the `#`, it would help readability to keep consistent